packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -132,3 +134,31 @@ String? activeDevToolsServerAddress;
 
 /// The uri for the connected vm service protocol.
 String? connectedVmServiceUri;
+
+/// If memory allocations are enabled, dispatch object creation.
```

Comment:
nit: are you trying to say "If memory allocation tracking is enabled"? We always allocate memory, so it's not clear what "memory allocations are enabled" means.